forum/models.py

```diff
@@ -44,6 +44,9 @@ def clean(self):
             return True
 
         raise ValidationError('Thread: must be only one visibility')
+    
+    def get_absolute_url(self):
```

Comment:
Is it used ?

Reply:
Yes, in order to redirect the user to the thread after a reply, otherwise the user has a blank page and has to refresh.